clientgen/template.gotpl

```diff
@@ -37,6 +37,14 @@ type {{ .Query.Name | go }} {{ .Query.Type | ref }}
     type  {{ .Name | go  }} {{ .Type | ref }}
 {{- end }}
 
+{{- if .Operation }}
+	type ClientInterface interface {
```

Comment:
this is pretty good !. but, I want to change the name of the interface. so, add setting for interface name in config.

Reply:
can you test if the latest one works? thanks for reviewing :)

Reply:
nice! please update example of example/github

Reply:
done :slightly_smiling_face: let's get this merged :feelsgood: